src/config/detector-hints.config

```diff
@@ -90,7 +90,7 @@ TARGET
 html
 
 MATCH
-link[href*="/Mvc/FrameworkContent/Styles/LandingMidnight.min.css?v=71278"]
+link[href*="/Mvc/FrameworkContent/Styles/LandingMidnight.min.css?v=72248"]
```

Comment:
If version changes maybe change to:
`link[href*="/Mvc/FrameworkContent/Styles/LandingMidnight.min.css?"]`
`*=` means it looks for this part of text.

Reply:
Thanks, should be good to go now